packages/ui-kit/src/Divider/style.module.css

```diff
@@ -0,0 +1,102 @@
+.divider {
```

Comment:
in this file prettier reformats all the tabulation on save
we need to have one convention in team, not to have conflicts

Reply:
Looks like we need a dedicated document where we can specify code standarts. Or at least set up linter settign to highlight tabulation issue (similar as we have in DN2). Also I will change my IDE settings to set tabulation to 2 spaces in CSS )

Reply:
Such little code style things like tabulation (tabWidth: 2) specified in prettier already, it should work on autosave in IDE. 
By the way in DN we run it on lintStage, if you don't run prettier on autosave. It looks like we need to add the same option in this repo.